### PR TITLE
Update csv & bundle with latest crd changes

### PR DESCRIPTION
**Pull request by abaiken**
Update the clusterserviceversion and costmanagment crd with latest changes before cutting a release

### CODE REVIEW

File: deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -86,7 +96,115 @@ spec:
...
+        kind: Secret
+        apiVersion: v1
+        metadata:
+          name: auth-secret-name
+          namespace: openshift-metering
+          annotations:
+            kubernetes.io/service-account.name: cost-mgmt-operator
+        data:
+          username: >-
+            Zm9v
```
Comment (chambridge) [inline deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml:129]:
```suggestion
            Y2xvdWQucmVkaGF0LmNvbSB1c2VybmFtZQ==
```

File: deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -86,7 +96,115 @@ spec:
...
+          password: >-
+            Zm9v
```
Comment (chambridge) [inline deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml:131]:
```suggestion
            Y2xvdWQucmVkaGF0LmNvbSBwYXNzd29yZA==
```

File: deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml [hunk 3]
```diff
@@ -86,7 +96,115 @@ spec:
...
+          token: >-
+            Zm9v
```
Comment (chambridge) [inline deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml:133]:
```suggestion
            Y2xvdWQucmVkaGF0LmNvbSB0b2tlbg==
```
Are we basically requiring that the user pull the JSON value from the pull secret .dockerconfigjson auths.cloud.openshift.com.auth?

I'm not sure the doc below covers that. 

I think we'd want something like:
```
oc get secret pull-secret -o "jsonpath={.data.\.dockerconfigjson}" | base64 --decode | jq '.auths."cloud.openshift.com".auth'
```

Comment (abaiken) [reply deploy/olm-catalog/cost-mgmt-operator/0.0.1/cost-mgmt-operator.v0.0.1.clusterserviceversion.yaml:133]: Great point. I will add this!!

**Pull request merged**